var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -115,6 +116,10 @@ def cmake_args(self):
     def setup_run_environment(self, env):
         env.prepend_path("PYTHONPATH", self.prefix.python)
         env.prepend_path("LD_LIBRARY_PATH", self.spec["podio"].libs.directories[0])
+        if "+sio" in self.spec and self.version >= Version("0.16"):
```

Comment:
FYI. We prefer using `self.spec.satisfies()` for non-variant checks, such as version (ranges).  (See https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints.)